README.md

```diff
@@ -58,6 +58,32 @@ make task
 - for every 'extra'(not correct) smell -> -1 point.
 - no negative points
 
+## Required JSON format to add new quiz
+```json
+{
+    "smell_types": ["Unnecessary Comment", "Logic Error", "Incorect Typing"],
+    "files": [
+        {
+            "file_id": "path",
```

Comment:
Wouldn't file_path instead of file_id bring more information what should be passed in request?

Reply:
Agree

Reply:
Yeah, sure